README.adoc

```diff
@@ -343,76 +384,20 @@ For more details about each of these REST endpoints and how you can tune their s
 with an `application.properties` file (in `src/main/resources`), see the
 the http://docs.spring.io/spring-boot/docs/{spring_boot_version}/reference/htmlsingle/#production-ready-endpoints[documentation about the endpoints].
 
-== View Spring Boot's Starters
-
-You have seen some of
-http://docs.spring.io/spring-boot/docs/{spring_boot_version}/reference/htmlsingle/#using-boot-starter[Spring Boot's "`starters`"].
-You can see them all
-https://github.com/spring-projects/spring-boot/tree/main/spring-boot-project/spring-boot-starters[here in source code].
-
-== JAR Support and Groovy Support
-
-The last example showed how Spring Boot lets you wire beans that you may not be aware you
-need. It also showed how to turn on convenient management services.
-
-However, Spring Boot does more than that. It supports not only traditional WAR file
-deployments but also lets you put together executable JARs, thanks to Spring Boot's loader
-module. The various guides demonstrate this dual support through the
-`spring-boot-gradle-plugin` and `spring-boot-maven-plugin`.
-
-On top of that, Spring Boot also has Groovy support, letting you build Spring MVC web
-applications with as little as a single file.
-
-Create a new file called `app.groovy` and put the following code in it:
-
-====
-[source,java]
-----
-@RestController
-class ThisWillActuallyRun {
-
-    @GetMapping("/")
-    String home() {
-        return "Hello, World!"
-    }
-
-}
-----
-====
-
-NOTE: It does not matter where the file is. You can even fit an application that small
-inside a https://twitter.com/rob_winch/status/364871658483351552[single tweet]!
+== Build an executable JAR or WAR
 
-Next, https://docs.spring.io/spring-boot/docs/{spring_boot_version}/reference/htmlsingle/#getting-started-installing-the-cli[install Spring Boot's CLI].
+As you've already seen, you can run the application from the command line by executing `./gradlew bootRun`. You can also build a single executable JAR file that contains all the necessary dependencies, classes, and resources. Building an executable jar makes it easy to ship, version, and deploy the service as an application throughout the development lifecycle, across different environments, and so forth.
```

Comment:
Should probably add mvn springBoot:run as well as ./gradlew

Reply:
Yes. Again, we can't toss out Maven.